hazelcast/src/main/java/com/hazelcast/internal/serialization/impl/FactoryIdHelper.java

```diff
@@ -143,6 +143,9 @@
     public static final String RELIABLE_ID_GENERATOR_DS_FACTORY = "hazelcast.serialization.ds.reliable_id_generator";
     public static final int RELIABLE_ID_GENERATOR_DS_FACTORY_ID = -46;
 
+    public static final String SPLIT_BRAIN_MERGE_POLICY_DS_FACTORY = "hazelcast.serialization.ds.split.brain.merge.policy";
```

Comment:
Yeah, I don't think we do this on any of those ID and hook classes, since the meaning is quite clear internally. If we do a cleanup here, it should be a separate PR though, this one is already a monster.

Reply:
:+1: